app/dashboard/views/fido_setup.py

```diff
@@ -81,9 +81,10 @@ def fido_setup():
         RP_ID,
         fido_uuid,
         current_user.email,
-        current_user.name,
+        current_user.name if current_user.name else current_user.email,
         False,
         attestation="none",
+        user_verification="discouraged",
```

Comment:
I'm not sure to see how this is linked to iOS ... Can you elaborate or send a link about this topic?

Reply:
https://developers.yubico.com/WebAuthn/Supporting_U2F_or_FIDO2_Security_Keys_on_iOS_or_iPadOS/Known_Issues.html

Reply:
I see, thanks!